README.md

```diff
@@ -1 +1,87 @@
+  - i.e. Georgia is a Lion  who is 4 years old
+1. create the schema from the migration by running `rails db:migrate`
+
+**Verify**:
+1. Check the `schema.rb` file and ensure all fields you expect are in the animals table
+  - if they are not, you may need to create another migration, or reset the database by running `rails db:reset`
+1. Run the `rails console` from the command line and create a few new animals and save them to the database. These animals will be used in the next set of requirements
+
+### View All Animals
+**Think**:
+- Which route (including controller#action) do you use for viewing _all_ of a given resource?
+- Which _model method_ do you use to retrieve _all_ of the items from the database?
+
+**Create**:
+1. the route
+1. the controller action which will retrieve all animals from the database
+1. the view containing HTML & ERB to see all animal data
+
+**Verify**:
+1. http://localhost:3000/animals should show a list of all animals in the database that you added in the `rails console` from the previous section
+
+
+### View One Animal
+**Think**:
+- Which route (including controller#action) do you use for viewing a _single_ resource?
+- Which _model method_ do you use to retrieve a _single_ item from the database?
+
+**Create**:
+1. the route (including the param)
+1. the controller action which will retrieve one specific animal from the database
+1. the view containing HTML & ERB to see a single animal details
+  - you can use [lorem pixel](http://lorempixel.com) to get animal images if you'd like to include a random photo for each animal
+1. add a link to the list of animals which will take you to each animals `show` page
+
+**Verify**:
+1. http://localhost:3000/animals will now include links for each animal's show page
+1. http://localhost:3000/animals/1 should show you the details of one specific animal (assuming you have an animal in the database with an ID of 1)
+
+### Create an Animal
+**Think**:
+- Which **routes** (including controller#action) do you use for creating a new resource? What is the purpose of each route and how do they work together to complete the whole action?
+- Which _model methods_ do you use to create and save a new item to the database?
+
+**Create**:
+1. the routes
+1. a link on the home page to create a new animal
+1. the controller action for the first route which renders the form
+1. the view to render the form
+1. the controller action for the second route which saves the form database
+1. the special controller method for utilizing strong parameters
+
+**Verify**:
+1. http://localhost:3000/animals should contain a button to create a new animal
+1. http://localhost:3000/animals/new should show the new animal form
+1. submitting the new animal form should save an animal to the database and show the full list, including the new animal on http://localhost:3000/animals
+
+
+
+## Optional
+- Complete the edit/update action
+- Complete the delete action
```

Comment:
Maybe add:
- Add an image field for each animal which is the location of an image on the internet and display the image on the animal's show page.